### PR TITLE
Added to calendar docs

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR adds the global edgeSize and icon color to calendar docs  
Will take care of Heading in another PR 
#### Where should the reviewer start?
calendar/doc.js
#### What testing has been done on this PR?
storybook visually tested the global edgesize and icon color to see changes 
#### What are the relevant issues?
related to issue #2802 
#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: src/js/utils/themeDocUtils.js [hunk 1]
```diff
@@ -74,6 +74,14 @@ export const themeDocUtils = {
       defaultValue: 'focus',
     },
   },
+  iconColor: {
```
Comment (ShimiSun) [inline src/js/utils/themeDocUtils.js]: why should `iconColor` be part of `themeDocUtils.js`? is it being used for other components?

File: src/js/utils/themeDocUtils.js [hunk 2]
```diff
@@ -74,6 +74,14 @@ export const themeDocUtils = {
...
+    'global.colors.icon': {
+      description: 'The color of a given icon.',
+      type: 'string | { dark: string, light: string }',
+      defaultValue: 'dark: #f8f8f8, light: #666666',
```
Comment (ShimiSun) [inline src/js/utils/themeDocUtils.js]:
```suggestion
      defaultValue: '{ dark: #f8f8f8, light: #666666 }',
```

File: src/js/utils/themeDocUtils.js (hunk 1 above)
Comment (britt6612) [reply src/js/utils/themeDocUtils.js]: Its being used in Carousel so I can go in and change that to use the themeDocUtils I believe those those are the only two using it Carousel and Calendar

Comment (ShimiSun) [reply src/js/utils/themeDocUtils.js]: Sounds good, let's do it then.

Comment (britt6612) [reply src/js/utils/themeDocUtils.js]: Ill do in another PR 👍

Comment (ShimiSun) [reply src/js/utils/themeDocUtils.js]: I think you can add it on this PR, so the generic iconColor will be used for more than a single component.

**Pull request merged**